pages/common/upt.md

```diff
@@ -0,0 +1,38 @@
+# upt
+
+> Unified interface for managing packages across various operating systems, like Windows, many Linux distributions, Android, macOS, FreeBSD and even Haiku.
+> It requires the OS package manager to be installed.
+> See also: `flatpak`, `brew`, `scoop`, `apt`.
+> More information: <https://github.com/sigoden/upt>.
+
+- Update the list of available packages:
+
+`upt update`
+
+- Search for a given package:
+
+`upt search {{search_term}}`
+
+- Show information for a package:
+
+`upt info {{package}}`
+
+- Install a given package:
+
+`upt install {{package}}`
+
+- Remove a given package:
+
+`upt {{remove|uninstall}} {{package}}`
+
+- Upgrade all installed packages:
+
+`upt upgrade`
+
+- Upgrade a given package:
+
+`upt upgrade {{package}}`
+
+- List installed packages:
```

Comment:
```suggestion
- List all installed packages:
```

Matching the wording from a previous example description.

Reply:
Well, "all" in the previous example description distinguishes `upt upgrade` from `upt upgrade {{package}}`. In this case, I don't think "all" is necessary.